src/main/java/entities/GuestUser.java

```diff
@@ -0,0 +1,8 @@
+package entities;
+
+public class GuestUser extends User{
+
+    public GuestUser() {
+        super();
+    }
```

Comment:
User should extend guest user since Guest User can only login and register, and User has more functionality